common/src/lib.rs

```diff
@@ -23,42 +26,27 @@ pub mod eqs {
 */
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct ZKStackEqProofInput {
-    #[serde(rename = "blob_data")]
-    pub data: Vec<u8>,
-
-    #[serde(rename = "blob_namespace")]
-    pub namespace_id: Namespace,
-
-    #[serde(rename = "nmt_multiproofs")]
-    pub share_proofs: Vec<NamespaceProof>,
-
-    #[serde(rename = "row_root_multiproof")]
-    pub row_proof: RowProof,
-
-    #[serde(rename = "blob_author")]
-    pub author: Option<AccAddress>,
-
-    pub data_root: [u8; 32],   // already matches
-    pub keccak_hash: [u8; 32], // already matches
-    // batch_number and chain_id are passed through to prevent proofs from being replayed
+    pub share_proof: ShareProof,
+    pub share_version: bool,
+    pub tail_padding: usize,
+    pub data_availability_root: [u8; 32],
```

Comment:
AFAIU we only need serde rename for json to test things, so arbitraty names are OK for that